Makefile

```diff
@@ -24,39 +24,40 @@ init:
 	@echo "'make init' is no longer needed"
 
 clean:
-	$(RM) -rf build docs/_build docs/api htmlcov
+	$(RM) -rf build docs/_build docs/api htmlcov ctapipe.egg-info dist
 	find . -name "*.pyc" -exec rm {} \;
 	find . -name "*.so" -exec rm {} \;
 	find . -name __pycache__ | xargs rm -fr
 
 test:
-	$(PYTHON) setup.py test
+	pytest
 
 doc:
-	cd docs && $(MAKE) html
+	cd docs && $(MAKE) html SPHINXOPTS="-W --keep-going -n --color -j auto"
```

Comment:
With this change, can we change the CI to only `make doc`, i.e. without additionally specifying SPHINXOPTS *again*?

Reply:
yes